### PR TITLE
Ft reset password 159961950

### PR DESCRIPTION
**Pull request by Tittoh**
#### What does this PR do?
Allow user to reset password

#### Description of Task to be completed?
- send password reset link to email
- Change password if the email is valid

#### How should this be manually tested?
```
git clone git@github.com:andela/ah-jarvis-frontend.git  -b ft-reset-password-159961950

cd ah-jarvis-frontend

yarn 

yarn sass:build

yarn start
```
- navigate to http://localhost:3000/reset/password and enter a valid email
- a link will be sent to the email.
- click on the link and it will redirect to http://localhost:3000/verify/<token><email>
- enter new password

#### What are the relevant pivotal tracker stories?
[#159961950 ](https://www.pivotaltracker.com/story/show/159961950) Users should be able to be presented with a reset password link

#### Screenshots
screen shot 2018-09-11 at 15 36 51
screen shot 2018-09-11 at 15 37 15
screen shot 2018-09-11 at 15 37 40
screen shot 2018-09-11 at 15 38 30

### CODE REVIEW

File: src/components/InputField/index.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ const InputField = ({
       <input
         type={type}
         name={name}
-        className={`validate ${failure ? 'invalid' : ''}`}
+        className={`validate ${failure ? 'text-red' : ''}`}
```
Comment (yeenab) [inline src/components/InputField/index.js]: Use the && operator here.

Sth like ```{`validate ${failure && 'text-red' }`}```

File: src/containers/ForgotPassword/index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+  render() {
+    const {
+      errors, failure, isFetching, success,
+    } = this.props.forgotPass;
+    let output;
+
+    if (success) {
+      output = (
+        <div className="row">
```
Comment (yeenab) [inline src/containers/ForgotPassword/index.js]: Can you make this one component, please? this is a lot of code being repeated

File: src/routes.js [hunk 3]
```diff
@@ -17,7 +19,8 @@ export default () => (
         <Route exact path="/" component={Home} />
         <Route exact path="/signin" component={Signin} />
         <Route exact path="/login" component={SocialAuth} />
-
+        <Route exact path="/reset/password" component={ForgotPasssword} />
```
Comment (yeenab) [inline src/routes.js]: Take advantage of the new file that contains Routes by Korir.
Include all routes in a central point

File: src/styles/css/index.css [hunk 4]
```diff
@@ -228,10 +228,26 @@ form .row {
 textarea.materialize-textarea {
   height: 7rem; }
 
+<<<<<<< HEAD
```
Comment (yeenab) [inline src/styles/css/index.css]: These are merge conflicts 😕

**Review by yeenab — CHANGES_REQUESTED** [review]
### Changes

File: src/containers/ForgotPassword/index.js (hunk 2 above)
Comment (Tittoh) [reply src/containers/ForgotPassword/index.js]: I'll work on it.

File: src/components/InputField/index.js (hunk 1 above)
Comment (Tittoh) [reply src/components/InputField/index.js]: I'm working on it

**Comment by yeenab** [comment]
@Tittoh Conflicts ooo

File: src/styles/scss/base/_base.scss [hunk 5]
```diff
@@ -70,15 +70,36 @@ form {
 textarea.materialize-textarea {
   height: 7rem;
 }
+<<<<<<< HEAD
```
Comment (yeenab) [inline src/styles/scss/base/_base.scss]: Remove these merge conflicts please 😢

File: src/routes.js [hunk 6]
```diff
@@ -19,7 +21,8 @@ export default () => (
         <Route exact path="/signin" component={Signin} />
         <Route path="/signup" component={SignUp} />
         <Route exact path="/login" component={SocialAuth} />
-
+        <Route exact path="/reset/password" component={ForgotPasssword} />
```
Comment (yeenab) [inline src/routes.js]: Can you move this in the Route file, please?

**Review by yeenab — CHANGES_REQUESTED** [review]
### Some Changes

File: src/routes.js (hunk 6 above)
Comment (yeenab) [reply src/routes.js]: I'm guessing if you actually updated develop recently you should have a file under `src/utils/routes.js` that can contain all the routes and you can just call them by `ROUTES.home` for `/`

**Review by yeenab — APPROVED** [review]
LGTM

**Pull request merged**